### ISSUE TITLE
doesn't support sound null safety

### ISSUE THREAD

**Issue by AmirAghaee** [mahdyar/persian-fonts-flutter#2]
Hi
After installation, I get this error:
```
Error: Cannot run with sound null safety, because the following dependencies
don't support null safety:

package:persian_fonts
```

It would be great if you could add null safety support to this package!
Thanks

**Comment by mahdyar** [comment]
Hi @AmirAghaee and thank you for your report!
Support for null safety is now implemented and is available since version `1.0.4`.

`persian_fonts 1.0.4` with null safety support is now available on pub.dev.

To upgrade:
```bash
flutter pub upgrade --null-safety
```

You can also upgrade it manually by editing `pubsec.yaml`:
```yaml
...
  persian_fonts: ^1.0.4
...

**Comment by AmirAghaee** [comment]
Thanks a lot!
